presto-product-tests/bin/run_on_docker.sh

```diff
@@ -102,7 +102,7 @@ function stop_docker_compose_containers() {
     stop_application_runner_containers ${ENVIRONMENT}
 
     # stop containers started with "up", removing their volumes
-    environment_compose down -v
+    environment_compose down -v || true
```

Comment:
Why this is needed? Is there a legitimate reason to accept not gracefully shut down sql server container?